### DISCUSSION TITLE
Tasmota WebUi: How to turn on and off using MQTT?

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#18262] [Q&A] [answered]
Hello, I have been successfully using all my custom Tasmota devices (Room switches that turn on/off lights and fans) for about 2 years. Thanks to the help I received in this forum everything is running smooth. I regularly update the firmware and currently I am running Tasmota 12.4.0 on all. Recently I installed Home Assistant on Raspberry Pi and using cloud MQQT server I was able to control all the devices on Home assistant. Now I would like to turn Tasmota WebUI interface off on all the devices, but I cant figure out the MQTT command to Turn on the Tasmota WebUI. I can turn it off by unchecking the HTTP API box under configure other option in WebUI. My Questions are: (a) What is the command to turn on the WebUi using MQTT. (b) Can I use home assistant dashboard to turn the WebUI on and off as needed instead of a MQTT client?

**Comment by sfromis** [comment, marked as answer]
Unchecking the HTTP API box (SetOption128) is not turning off the web UI as such, only some types of usage. You can turn on/off the web UI (or restrict to admin mode) using the command [WebServer](https://tasmota.github.io/docs/Commands/#webserver), and all commands are [available via MQTT](https://tasmota.github.io/docs/MQTT/#commands-over-mqtt).

Using HA to issue commands to Tasmota is just another MQTT client.

**Reply by NsinghP** [reply]
Thank you. I had asked similar question 2 years ago and now I looked at that post and  figured out how to use webserver command, I had just forgotten how to use it over MQTT. Thankyou for your help. Here is the old post: [https://github.com/arendst/Tasmota/discussions/12333](url).
